script.js

```diff
@@ -1,28 +1,92 @@
+$(document).on("ready", function() {
+
 var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
-  deck: [],
-  players: [],
-  hands: [],
-  buildDeck: function(){
+    deck: [],
+    players: [],
+    hands: [],
+
+    buildDeck: function(){
+        for (i=0; i<values.length; i++) {
+            game.deck.push(values[i]+suits[0]);
+        }
+        for (i=0; i<values.length; i++) {
+            game.deck.push(values[i]+suits[1]);
+        }
+        for (i=0; i<values.length; i++) {
+            game.deck.push(values[i]+suits[2]);
+        }
+        for (i=0; i<values.length; i++) {
+            game.deck.push(values[i]+suits[3]);
+        }
+
```

Comment:
I tend to favor `forEach` loops over `for` loops when possible...but you could simplify this a little with nested loops using either method, i.e.

```js
var self = this
values.forEach(function(value, index) {
  suits.forEach(function(suit) {
    self.deck.push({
      value: value,
      suit: suit[0,1] // really only need the first character
     worth: index // uses a value's index to determine card's worth - useful for easy sorting
   })
  }
}
```